_vendors/sendgrid.yaml

```diff
@@ -0,0 +1,8 @@
+---
+base_pricing: $19.95 per month
+name: Sendgrid
+percent_increase: 450%
```

Comment:
_increase_
So $90-20 is a  $70 increase
$70/20 = 3.5 x increase = 350% increase